### ISSUE TITLE
3.6.7 still has ghost image

### ISSUE THREAD

**Issue by atawfique21** [michelecocuccio/simple-react-lightbox#165]
**Describe the bug**
This is a follow up to closed ticket: #163
Over the past few days, I've been trying to fix the same issue but running into problems. I've included my environment again, this time with node_modules/simple-react-lightbox. I was on `v3.6.6` of SRL as of ticket #163 but updating to `3.6.7` using `npm update simple-react-lightbox` and restarting `next dev` did not fix the issue. I also tried running `npm update` as a whole and there does seem to be some related errors with `react-zoom-pinch-pan v1.6.1` which I left as a comment here: https://github.com/michelecocuccio/simple-react-lightbox/issues/163#issuecomment-899143864. It seems `react-zoom-pinch-pan v1.6.1` has outdated dependencies and perhaps bumping this package will fix the issue? 

**Environment*
`"next": "^11.1.0"
"react": "17.0.2"`
```
"node_modules/simple-react-lightbox": {
      "version": "3.6.7",
      "resolved": "https://registry.npmjs.org/simple-react-lightbox/-/simple-react-lightbox-3.6.7.tgz",
      "integrity": "sha512-qPZO8LhZncmaXumk1ze5cQTq/I/ovNZ53Ly05U+m/f3hqbTGjWdKqK//2vTb0wlFGE8CFY0/XgT7eUd6TwPDSg==",
      "dependencies": {
        "body-scroll-lock": "^3.1.5",
        "fast-deep-equal": "^3.1.3",
        "framer-motion": "^2.9.4",
        "lodash": "^4.17.20",
        "react-swipeable": "^6.0.0",
        "react-use": "^15.3.4",
        "react-zoom-pan-pinch": "^1.6.1",
        "rollup-plugin-node-polyfills": "^0.2.1",
        "subscribe-event": "^1.1.1",
        "use-debounce": "^5.1.0"
      },
      "peerDependencies": {
        "react": "^17.0.1",
        "react-dom": "^17.0.1"
      }
    }
```

Thank you very much!

**Comment by michelecocuccio** [comment]
Hello, I don't have any issues with Next.js ghost images. The "ghost" PNG and SVG coming from the Next.js have been excluded and they are not taken in consideration when the lightbox start. I have tested it in three different demos with Next.js and I can't reproduce the issue. I have released a new version just now (3.6.8) where I updated some outdated dependencies. Can you let me know if you still have issue? Thanks

**Comment by atawfique21** [comment]
Hi there,

I updated the package to 3.6.8 and when running `npm update`, there were no dependency errors present. Thank you for resolving that issue. However, the 'ghost' image is still present and the link to the image remains the same:
`data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQ0IiBoZWlnaHQ9IjE0NCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4=`

I have tried deleting package_lock.json, node_modules and running `npm i` again. 
Is there a way to just hardcode SRL to make sure this exact image is not included in SRL's lightbox? I understand this may be frowned upon but this SVG should never be present in SRL's context.

Let me know what you think.
Thank you for your responsiveness and support.

**Comment by michelecocuccio** [comment]
@atawfique21 I am well aware what the issue is, and I know the this SVG should never be present. I am working to solve this issue as Next.js has recently changed its component again. Thanks

**Comment by michelecocuccio** [comment]
@atawfique21 I have released version 3.6.9-0. It should exclude all the image with "aria-hidden" so it should automatically exclude any Next.js ghost image. Please let me know.

**Comment by atawfique21** [comment]
Great work. Version 3.6.9-0 fixed the issue!

Thanks!
